components/media-upload-button/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+import { __ } from 'i18n';
+import Button from 'components/button';
+
+class MediaUploadButton extends Component {
```

Comment:
Yes, I hesitated but I decided to put it here because it can be useful outside the Editor (in other parts of WordPress).